manual/manual-fr.adoc

```diff
@@ -0,0 +1,685 @@
+
+=== Quitter le jeu
+
+Lorsque vous aurez fini de jouer à Freedoom, appuyez sur _Échap_ pour afficher le menu principal et sélectionnez _Quit Game_ pour quitter. Vous aurez surement envie de sélectionner _Save Game_ en premier afin d'enregistrer votre progrès et pouvoir revenir à l'endroit où vous vous étiez arrêté lors de votre prochaine session.
+
+=== Raccourcis clavier
+
+Quelques raccourcis clavier pour accéder aux fonctions du menu instantanément.
+
+[cols="4,6,15",width="90%",align="center",valign="middle"]
+|==========================
+| **Échap** | <<menus,Menu>> | Affiche le menu principal.
+| **F1** | Info | Affiche l'écran d'aide qui présente des informations sur les objets dans le jeu.
+| **F2** | <<savegame,Sauvegarder>> | Affiche le menu _Save Game_.
+| **F3** | <<savegame,Charger>> | Affiche le menu _Load Game_.
+| **F4** | Volume | Affiche le menu pour régler le volume.
+| **F6** | <<savegame,Sauvegarde Rapide>> | Savevarge la partie à l'emplacement de _sauvegarde rapide_, ce qui permet de gagner du temps si vous sauvegardez fréquemment lors d'une partie.
+| **F7** | Quitter la Partie | Termine la partie actuelle et retourne au menu principal.
+| **F8** | Messages | Bascule entre montrer ou masquer les messages à l'écran pour les objets ramassés, les clés, les triches, etc...
+| **F9** | <<savegame,Charger Sauvegarde Rapide>> | Charge la partie de votre emplacement _sauvegarde rapide_.
+| **F10** | Quitter le Jeu | Quitte le jeu et vous retourne au système d'exploitation.
+| **F11** | Luminosité | Permet de régler le niveau de luminosité dans le jeu.
+|==========================
+
+<<<
+
+== Comment Jouer
+
+image::images/map01-sshot.png[Freedoom Screenshot,width="640",pdfwidth="70vw",align="center"]
+
+Freedoom est un jeu de tir à la première personne (FPS) en temps réel. Explorez une série de niveaux dans lesquels devez trouver le chemin vers la sortie. Un assortiment de monstres tentera de vous arrêter, et vous aurez besoin d'utiliser les armes à votre disposition pour vous défendre. Certaines parties des niveaux peuvent demeurer inaccessibles jusqu'à ce que vous trouviez une clé spécifique ou que vous trouviez un interrupteur pour ouvrir un passage. Cela signifie que le gameplay impliquera des puzzles d'exploration pour dénicher des objets cachés ainsi que des puzzles d'action en temps réel sur le placement et le timing de vos tirs.
+
+Voici un tableau avec les contrôles de base du jeu. **Les défauts de Doom sont largement considérés suboptimaux;** vérifiez votre portage pour savoir comment les reconfigurer. Des alternatives courantes sont fournies, mais il n'y a pas de "meilleure" solution qui convient pour tout le monde -- vous allez peut-être devoir expérimenter. Au minimum, vous devez être à l'aise en vous déplaçant dans l'une des quatre directions tout en tournant et en tirant simultanément.
```

Comment:
```suggestion
Voici un tableau avec les contrôles de base du jeu. **Les défauts de Doom sont généralement considérés comme obsolètes;** vérifiez votre portage pour savoir comment les reconfigurer. Des alternatives courantes sont fournies, mais il n'y a pas de « meilleure » solution qui convient pour tout le monde -- vous allez peut-être devoir expérimenter. Au minimum, vous devez être à l'aise en vous déplaçant dans l'une des quatre directions tout en tournant et en tirant simultanément.
```

Reply:
I'd change `les valeurs par défaut` to `les contrôles par défaut`. Aside from that, this is better.